### ISSUE TITLE
decode fluorescence sequences

### ISSUE THREAD

**Issue by RensDimmendaal** [songlab-cal/tape#35]
I'm trying to decode the sequences in the fluorescence benchmark dataset, but  when I use the default tokenizer it returns nonsensical sequences.
I've included an example below.

Can you tell us how to decode the sequences?

```python
import dotenv
import os
from pathlib import Path
import json


dotenv.load_dotenv()
data_path = Path(os.getenv("DATA_ROOT"))

task = "fluorescence"
fpath = data_path / f"tape/{task}/{task}_train.json"
with open(fpath,'r') as f:
    train = json.load(f)
    
from tape import TAPETokenizer
tokenizer = TAPETokenizer(vocab='iupac')

''.join(tokenizer.convert_ids_to_tokens(train[0]['primary']))
```

```
>>> 'RIFDDKESFUUOHKUDKCFCUMFGIERURFDFDFC<unk>SXFIKSKIEHBSSFIKOUOVOSKUSSKRXFUPBERQXOCGLIPGCEEIR<unk>LODFXUPDQSHEEICCFMXISQ<unk>DUIEDFCSKUMQHDKIFHCEIDCFMHKFGIKDXMXMRGMUXHL<unk>CIPIMFHIUMEIHQGIHDCFRUPK<unk>CGXPPMSOHFCFOUKKOCMGXKRSPR<unk>KRICOMDIQCGLUKKDEUS<unk><unk>FHSGFLCDQXI'
```

**Comment by rmrao** [comment]
Our raw JSON data is still in our old tensorflow encoding it looks like. I will update this.

**Comment by rmrao** [comment]
Just uploaded the new data and fixed the links, so this should be fixed.

**Comment by rmrao** [comment]
Fixed by #41

**Comment by RensDimmendaal** [comment]
Thanks a lot @rmrao 

There still steems to be an issue with the test sets in json format. At least fluorescence and stability have some unrecognized characters at the start which makes loading them an issue.

Code to replicate:

```
data_path="."

curl http://s3.amazonaws.com/proteindata/data_raw_pytorch/fluorescence.tar.gz -o "$data_path/fluorescence.tar.gz"
tar -xf "$data_path/fluorescence.tar.gz" --directory "$data_path/"

zless fluorescence_test.json
```
